airflow/providers/amazon/aws/example_dags/example_s3_to_redshift.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+This is an example dag for using `S3ToRedshiftTransfer` to copy a S3 key into a Redshift table.
+"""
+
+from os import getenv
+
+from airflow import DAG
+from airflow.providers.amazon.aws.operators.s3_to_redshift import S3ToRedshiftTransfer
+from airflow.utils.dates import days_ago
+
+# [START howto_operator_s3_to_redshift_env_variables]
+S3_BUCKET = getenv("S3_BUCKET", "bucket")
+S3_KEY = getenv("S3_KEY", "key")
```

Comment:
Why are we relying on env vars, rather Airflow Variables+templates? `{{ var.s3_key}}`

Reply:
We have this in all of our system tests. So that we can easily pass them to the example dags. See https://github.com/apache/airflow/blob/master/TESTING.rst#environment-for-system-tests

Reply:
I think it makes sense to switch to Airflow Variables+templates like Ash said and set them via env variables: https://airflow.apache.org/docs/stable/concepts.html?highlight=xcom#storing-variables-in-environment-variables. 

So we only need to prefix all env vars with `AIRFLOW_VAR`.
For example: set `AIRFLOW_VAR_S3_KEY=key` in `variables.env` and access them via `{{ var.s3_key}}` here. WDYT @potiuk @turbaszek ?

Reply:
But how do we then define defaults in case these `var.`s are not set? Or should we make all vars required then?